dex/src/fees.rs

```diff
@@ -81,25 +76,20 @@ impl FeeTier {
     }
 
     #[inline]
-    pub fn maker_rebate(self, pc_qty: u64) -> u64 {
-        use FeeTier::*;
-        let rate: U64F64 = match self {
-            MSRM => rebate_bps(5),
-            Base | SRM2 | SRM3 | SRM4 | SRM5 | SRM6 => rebate_bps(3),
-        };
-        rate.mul_u64(pc_qty).floor()
+    pub fn maker_rebate(self, _pc_qty: u64) -> u64 {
+        0
```

Comment:
Maybe we use rebate_tenth_of_bps(0).mul_u64(pc_qty).floor(). Not sure why we used to add a 2^-64 rebate to every tier before, but can keep it just for consistency.